packages/sui-mono/bin/sui-mono-changelog.js

```diff
@@ -28,7 +30,7 @@ const folders = program.args.length ? program.args : getWorkspaces()
 const changelogOptions = {
   preset: 'angular',
   append: false,
-  releaseCount: 0,
+  releaseCount: 1,
```

Comment:
Instead of generating all the changelog let's generate only the part for the last release